plugins/reindexer/dao.go

```diff
@@ -153,8 +155,8 @@ func reindex(ctx context.Context, sourceIndex string, config *reindexConfig, wai
 			return nil, err
 		}
 
-		if destinationIndex == "" {
-			err = postReIndex(ctx, sourceIndex, newIndexName)
+		if !isClone {
+			err = postReIndex(ctx, sourceIndex, newIndexName, isClone)
```

Comment:
@ShahAnuj2610 you are using inversion of `isClone` to `isDelOldIndex` which makes things confusing

Instead of using boolean parameter use enum that is easy to read.